src/main/webapp/app/statistics/quiz-statistic/show-quiz-statistic.html

```diff
@@ -0,0 +1,200 @@
+                        display: true,
+                    },
+                }]
+            },
+            hover: {animationDuration: 0},
+            //add numbers on top of the bars
+            animation: {
+                duration: 500,
+                onComplete: function () {
+                    var chartInstance = this.chart,
+                        ctx = chartInstance.ctx;
+                    var fontSize = 12;
+                    var fontStyle = 'normal';
+                    var fontFamily = 'Calibri';
+                    ctx.font = Chart.helpers.fontString(fontSize, fontStyle, fontFamily);
+                    ctx.textAlign = 'center';
+                    ctx.textBaseline = 'middle';
+
+                    this.data.datasets.forEach(function (dataset, i) {
+                        var meta = chartInstance.controller.getDatasetMeta(i);
+                        meta.data.forEach(function (bar, index) {
+                            var data = (Math.round(dataset.data[index] * 100) /100);
+                            var dataPersentage = (Math.round((dataset.data[index] /barChartData.participants) *1000)/10);
+
+                            var position = bar.tooltipPosition();
+
+                            if(dataPersentage > 6) {
+                                if(position.y > 15) {
+                                    ctx.fillStyle = 'black';
+                                    ctx.fillText(data, position.x, position.y - 10);
+
+
+                                    if(barChartData.participants !== 0) {
+                                        ctx.fillStyle = 'white';
+                                        ctx.fillText(dataPersentage.toString()+"%", position.x, position.y + 10);
+                                    }
+                                }
+                                else{
+                                    ctx.fillStyle = 'white';
+                                    if(barChartData.participants !== 0) {
+                                        ctx.fillText(data + " / " + dataPersentage.toString()+"%", position.x, position.y + 10);
+                                    }else{
+                                        ctx.fillText(data, position.x, position.y + 10);
+                                    }
+                                }
+                            }
+                            else{
+                                ctx.fillStyle = 'black';
+                                if(barChartData.participants !== 0) {
+                                    ctx.fillText(data + " / " + dataPersentage.toString()+"%", position.x, position.y - 10);
+                                }else{
+                                    ctx.fillText(data, position.x, position.y - 10);
+                                }
+                            }
+                        });
+                    });
+                }
+            }
+        }
+    });
+
+</script>
```

Comment:
Same as the other HTML-File: Please move Javascript code to .js file and write comments in English.